src/stores/account.js

```diff
@@ -36,8 +41,24 @@ export const useAccountStore = defineStore({
     userId(state) {
       return state.account?.user_id
     },
-    isReviewPaymentRequired(state) {
-      return STATUS_CLIENT_REVIEW_PAYMENT_REQUIRED.includes(state.account?.status)
+    accountStatus(state) {
+      return state.account?.status
+    },
+    redirectToExternalBillingNeeded(state) {
+      return !state.account?.status || state.accountStatuses.REGULAR === state.account?.status
+    },
+    billingAccessPermitted(state) {
+      return [
+        state.accountStatuses.BLOCKED,
+        state.accountStatuses.DEFAULTING,
+        state.accountStatuses.TRIAL,
+        state.accountStatuses.ONLINE
+      ].includes(state.account?.status)
```

Comment:
evita criar um array e percorrer ele pra verificar o status
```suggestion
      return  !!state.accountStatuses[state.account?.status]
```

Reply:
@william-tome o array contem somente 4 status e o accountStatuses tem 5, por isso não temos como fazer a sugestão.

Reply:
Não tinha visto esse ponto.